airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1886,9 +1888,30 @@ definitions:
           - "{{ config['base_url'] or 'https://app.posthog.com'}}/api"
           - "https://connect.squareup.com/v2/quotes/{{ stream_partition['id'] }}/quote_line_groups"
           - "https://example.com/api/v1/resource/{{ next_page_token['id'] }}"
+      url:
+        sharable: true
```

Comment:
This is supposed to be `linkable` now, right?

Also, I don't think the `url` field should actually be linkable, because we now expect it to also include the endpoint path, and we don't expect that to be common across streams.

Reply:
That is a leftover from `un-merging` of the PoC PR,, fixed now.